### PR TITLE
Register clients in ClientCollection

### PR DESCRIPTION
**Pull request by amenophis**

### CODE REVIEW

File: src/ClientCollection.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Amenophis\Elasticsearch;
```
Comment (B-Galati) [inline src/ClientCollection.php]: Goes in the bridge no ?

File: src/ClientCollection.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+use Amenophis\Elasticsearch\Exception\ClientNotFound;
+use Elasticsearch\Client;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+class ClientCollection
```
Comment (B-Galati) [inline src/ClientCollection.php]: Is this `@internal` ?

File: src/ClientCollection.php (hunk 1 above)
Comment (amenophis) [reply src/ClientCollection.php]: Good catch !

File: src/ClientCollection.php (hunk 2 above)
Comment (amenophis) [reply src/ClientCollection.php]: yep

**Pull request merged**